main.tf

```diff
@@ -272,7 +273,7 @@ data "aws_iam_policy_document" "monitoring-rds-assume-role-policy" {
 
 resource "aws_iam_role" "rds-enhanced-monitoring" {
   count              = "${var.monitoring_interval > 0 ? 1 : 0}"
-  name_prefix        = "rds-enhanced-monitoring-${var.envname}-"
+  name_prefix        = "rds-enhanced-mon-${var.envname}-"
```

Comment:
Why the name change? While minor this would break backwards compatibility as the resource would be re-created for what would appear to be no reason.